flink-state-backends/flink-statebackend-forst/src/main/java/org/apache/flink/state/forst/ForStKeyedStateBackend.java

```diff
@@ -227,6 +233,21 @@ public void setup(@Nonnull StateRequestHandler stateRequestHandler) {
         this.stateRequestHandler = stateRequestHandler;
     }
 
+    @Override
+    public <N, S extends State, SV> S getOrCreateKeyedState(
```

Comment:
I'm not sure if we still need `createState` interface in case the `getOrCreateKeyedState` is provided. How about make that `protected`?

Reply:
👍 Methods in an interface are `public` and  `abstract`, `createState()` can't be marked as `protected `, so I removed `createState()` from `AsyncStateBackend`.